### PR TITLE
Task(array2)

### PR DESCRIPTION
**Pull request by Artofpaganini**
1)Even-odd quantity
2)largeNumbers
3)MinMax(float,double)
4)Sum and Sr Ar

### CODE REVIEW

File: Task2.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        int count = 0;
+        //инициализируем эл-ты массива
+        for (int index = 0; index < Mass.length; index++) {
+            Mass[index] = (float) (Math.random() * 100);
+            //вывод на экран эл-ов массива и их индексы
+            System.out.print(index + " ");
+            System.out.print(" ");
+            System.out.println(Mass[index] + " ");
+        }
+        System.out.println(" ");
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task2.java]: at seriously  for  me  it's  new one ! xD

File: Task3.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+public class Task3 {
+    public static void main(String[] args) {
+/*  Задан массив, состоящий из 16 элементов вещественного типа.
+          Определить
+           --индексы (местоположение) максимального и минимального элементов
+-создать массив вещ типа на 16 эл-ов
+-вывести на экран эл-ты и индексы
+-опред макс эл-т  и его индекс
+-вывести их на экран
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task3.java]: mistake =/

File: Task3.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                indexMin = index;
+            }
+        }
+        //вывод  мин эл-та  и его индекса
+        System.out.println("Минимальный э-т массива " + " " + valueMin);
+        System.out.println(" ");
+        System.out.println("Индекс мин э-та массива " + " " + indexMin);
+        System.out.println(" ");
+    }
+}
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task3.java]: i'm  rly happy  xD at least one  didn't failed

File: Task4.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            srArMass = sumMass / Mass.length;
+        }
+        //вывод на экран ср.арифм.
+        System.out.print(srArMass + " ");
+        System.out.println(" ");
+        //находим разность между суммой и ср. арифм. эл-ов массива
+        razEl = sumMass - srArMass;
+        System.out.print(razEl + " ");
+        //Создаем новый массив
+        int newMass[] = new int[razEl];
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task4.java]: i don't  know  xD
useless variable

File: Task3.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (mick842) [reply Task3.java]: Why do you afraid of mistakes?)
It's alternative and very powerful way to learn something.

File: Task2.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+public class Task2 {
+    public static void main(String[] args) {
+      /*  Задан массив,
+            --состоящий из 15 элементов вещественного типа.
+          Определить
+            --количество элементов,значения которых больше первого элемента.
+
+     -создать массив с 15 рандомными эл-тами типа double,float
+     -вывести на экран
+     -вывести на экран индекс каждого эл-та
```
Comment (mick842) [inline Task2.java]: Do you really need this step?)
Every time there will be the following sequence:
`0, 1, 2 ... Array.length - 1`

File: Task2.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+     -определить эл-ты  которые больше первого эл-та
+     -вывести на экран
+     -определить их индексы
```
Comment (mick842) [inline Task2.java]: I think it's kind of extra thing.
You can use just values.

File: Task2.java [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+            System.out.println(Mass[index] + " ");
+        }
+        System.out.println(" ");
+        float largeEl = 0;
+        int indexAll = 0;
+        //опред. эл-ты и их индексы которые больше первого
+        for (int index = 0; index < Mass.length; index++) {
+            if (Mass[index] > Mass[0]) {
+                largeEl = Mass[index];
+                indexAll = index;
```
Comment (mick842) [inline Task2.java]: It's a confusing variable name.

If you want to print current index of bigger than the first, you can use the variable `index` instead.

File: Task4.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        System.out.print("Cymma" + " " + sumMass + " ");
+        System.out.println(" ");
+        //находим ср. арифм  массива
+        for(int index = 0; index < Mass.length; index++){
+            srArMass = sumMass / Mass.length;
+        }
+        //вывод на экран ср.арифм.
+        System.out.print("Ср. арифм " + " " + srArMass + " ");
+        System.out.println(" ");
+        //находим разность между  эл-тами и ср. арифм. эл-ов массива
```
Comment (mick842) [inline Task4.java]: Yes, it is!

But could you please complete one more thing?)
You need to create a new array, but not just a print values.

File: Task2.java (hunk 5 above)
Comment (Artofpaganini) [reply Task2.java]: why not? when you make a compiler , you see table , where everything is understanding xD
That's only for  me and only for understanding )

File: Task2.java [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+        //создаем массив с 15 эл-ами вещ. типа
+        float Mass[] = new float[15];
+        float firstEl = Mass[0];
+        int count = 0;
+        //инициализируем эл-ты массива
+        for (int index = 0; index < Mass.length; index++) {
+            Mass[index] = (float) (Math.random() * 100);
+            //вывод на экран эл-ов массива и их индексы
+            System.out.print(index + " ");
+            System.out.print(" ");
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task2.java]: I think i carried away

File: Task2.java (hunk 7 above)
Comment (Artofpaganini) [reply Task2.java]: I agreed   with you , mistake

File: Task4.java [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        //находим сумму эл-ов массива
+        for(int index = 0; index < Mass.length; index++){
+            sumMass += Mass[index];
+
+        }
+        //вывод суммы эл-ов на экран
+        System.out.print(sumMass + " ");
+        System.out.println(" ");
+        //находим ср. арифм  массива
+        for(int index = 0; index < Mass.length; index++){
```
[the remark this answers is not in the document]
Comment (Artofpaganini) [reply Task4.java]: at seriously  at this moment  i thought that  i can't use the  length witout   cycle,  but  now  i 'm understanding  ,  that i can use it  )

File: Task4.java (hunk 8 above)
Comment (Artofpaganini) [reply Task4.java]: I make it! ;)  myself xd

**Pull request merged**